### DISCUSSION TITLE
Bundeling and serving static files

### DISCUSSION THREAD

**Discussion by dacianb** [pocketbase/pocketbase#1023] [Q&A]
A follow up question to https://github.com/pocketbase/pocketbase/discussions/879#discussion-4524782

Is there a way to bundle the static files (Vue, Svelte etc.) inside the GO app? Basically shipping a single bundle with everything inside.

**Comment by ganigeorgiev** [comment]
Yes, [go supports files embedding](https://pkg.go.dev/embed) out of the box, so there is nothing really PocketBase specific.

For an example you could check how we register the Admin UI in:
- https://github.com/pocketbase/pocketbase/blob/master/ui/embed.go
- https://github.com/pocketbase/pocketbase/blob/master/apis/base.go#L146-L153

Please note that, as mentioned in the linked #879, the second argument of `apis.StaticDirectoryHandler` has changed in v0.8.0-rc and now it is used to determine whether to fallback to index.html on missing static file (eg. suitable for SPA and/or frontend routers using pretty urls).
